plugins/social/src/main/java/de/unistuttgart/iaas/amyassist/amy/plugin/social/SocialLogic.java

```diff
@@ -76,6 +76,16 @@ public void init() {
 			this.pluginNames.put(pluginHumanReadableName, pluginInfo[i]);
 		}
 	}
+	
+	/**
+	 * receive a random name answer
+	 * 
+	 * @return her name
+	 */
+	protected String getMyNameIs() {
```

Comment:
`protected `?

Reply:
good question, I will ask @FlxB2 about it

Reply:
my bad, @Hobbitsloth could you change it to default?